src/Orleans.Runtime/Catalog/ActivationData.cs

```diff
@@ -456,7 +456,7 @@ public void SetCollectionTicket(DateTime ticket)
         /// </summary>
         internal bool IsUsingGrainDirectory => this.PlacedUsing.IsUsingGrainDirectory;
 
-        public Message Running { get; private set; }
+        public Message Blocking { get; private set; }
```

Comment:
I describe this to myself as "the message which is currently blocking non-interleaving messages from execution"

Reply:
This probably warrants a comment